pattern-1/tile/tile.yml

```diff
@@ -22,100 +22,195 @@
 # The high-level description of your tile.
 # Replace these properties with real values.
 #
-name: wso2is-tile
+name: wso2is-tile-pattern-1
```

Comment:
Should the pattern details be indicated in the name? The API Manager tile does not include pattern details.

Reply:
I have added it for our convenience as we are testing all these in the same environment. But from an end user's perspective, this is not needed. Therefore fixed with 2b61e3d